com.unity.cinemachine/CHANGELOG.md

```diff
@@ -5,10 +5,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
-## [2.10.4] - 2025-02-06
-- Bugfix: FramingTransposer with a dead zone would sometimes drift.
-- Cinemachine Shot Editor longer provides UX to create cameras when editing a prefab.
+## [2.10.4] - 2025-12-31
+
+### Unreleased
```

Comment:
I know this is temporary, but it feels like it's not the standard formatting for unreleased stuff. I don't remember what it should be, though.